pulpcore/pulpcore/app/models/task.py

```diff
@@ -69,9 +69,9 @@ def online_workers(self):
         """
         Returns a queryset of workers meeting the criteria to be considered 'online'
 
-        To be considered 'online', a worker should both have its 'online' field set to True
-        and have a recent heartbeat. "Recent" is defined here as "within the pulp process timeout
-        interval".
+        To be considered 'online', a worker must have a recent heartbeat timestamp and must not
+        have the 'gracefully_stopped' flag set to True. "Recent" is defined here as "within
```

Comment:
These docs duplicate the docs in the `online()` method which makes me concerned about maintaining them. Can these be removed and point at the `online()` via a Sphinx link instead?

Reply:
I'd prefer not to because, since the code also has to be mostly duplicated but isn't exactly the same, both will need to be refactored together in sync anyway, and fixing the docstrings would be part of that refactoring.

Reply:
I agree since the code is also duplicated and deduplicating that in this PR isn't the goal.